File: CPP/Manusia.cpp
/*Saya Adinda Salsabilla 2005319 mengerjakan Latihan 2 dalam mata kuliah Desain dan Pemrograman Berorientasi Objek 
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin*/

#include <iostream>
#include <string>
using namespace std;

// class Manusia sebagai parent (Superclass)
class Manusia{
    
    // private atribute
    private:
        string nik;
        string nama;
        string jenis_kelamin;

    public:

    Manusia(){ // constructor
    }

    /* SETTER & */

    // NIK
    void setNik(string nik){
        this->nik = nik;
    }

    string getNik(){
        return this->nik;
    }

    // NAMA
    void setNama(string nama){
        this->nama = nama;
    }

    string getNama(){
        return this->nama;
    }

    // JENIS KELAMIN
    void setJenisKelamin(string jenis_kelamin){
        this->jenis_kelamin = jenis_kelamin;
    }

    string getJenisKelamin(){
        return this->jenis_kelamin;
    }

    ~Manusia(){ // destructor
    }
};

File: PHP/Manusia.php
<!-- Saya Adinda Salsabilla 2005319 mengerjakan Latihan 2 dalam mata kuliah Desain dan Pemrograman Berorientasi Objek 
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin -->

<?php

// class Manusia sebagai parent (Superclass)
class Manusia{
    // private atribute
    private $nik;
    private $nama;
    private $jenis_kelamin;

    // constructor
    function __construct(){
    }

    /* SETTER & GETTER */
    
    // NIK
    function setNik($nik){
        $this->nik =$nik;
    }
    
    function getNik(){
        return $this->nik;
    }
    
    // NAMA
    function setNama($nama){
        $this->nama =$nama;
    }
    
    function getNama(){
        return $this->nama;
    }
    
    // JENIS KELAMIN
    function setJenisKelamin($jenis_kelamin){
        $this->jenis_kelamin =$jenis_kelamin;
    }
    
    function getJenisKelamin(){
        return $this->jenis_kelamin;
    }
 
    // destructor
    function __destruct(){
    }
}

?>